src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -43,9 +44,25 @@ class GroupManagerImpl(
     * Even though updates go through the workqueue, we want to make sure multiple readers always read
     * the latest version of the root. This could be solved by a @volatile too, but this is more explicit.
     */
-  private[this] val root = LockedVar(initialRoot)
-
-  override def rootGroup(): RootGroup = root.get()
+  private[this] var root = LockedVar(initialRoot)
+
+  override def rootGroup(): RootGroup =
+    root.get() match { // linter:ignore:UseGetOrElseNotPatMatch
+      case None =>
```

Comment:
In my view if would be great to have a test for this scenario too.

Reply:
I will add one.